examples/shader/compute_shader_game_of_life.rs

```diff
@@ -19,8 +19,10 @@ fn main() {
     App::new()
         .insert_resource(ClearColor(Color::BLACK))
         .insert_resource(WindowDescriptor {
-            // uncomment for unthrottled FPS
-            // vsync: false,
+            // uncomment one for unthrottled FPS
+            // if it doesn't work try the other one
+            //present_mode: PresentMode::Mailbox,
+            //present_mode: PresentMode::Immediate,
```

Comment:
How detailed would you like? :)
```suggestion
            // Uncomment one for unthrottled FPS.
            // Mailbox allows one framebuffer to be presented, one to be queue, and another to be rendered into.
            // Mailbox is not supported on all platforms (e.g. macOS) and will fall back to Fifo which allows one
            // frame to be presented, and one to be rendered or queued, but if one is queued, it must wait until the
            // frame being presented is swapped for the one that is queued, which prevents unthrottled FPS.
            // If using Mailbox does not work for you, try Immediate which will always allow obtaining a new frame to
            // render into, but can produce tearing artifacts where a frame is swapped while it is being displayed.
            //present_mode: PresentMode::Mailbox,
            //present_mode: PresentMode::Immediate,
```

Reply:
Yep, I'm on board with that :)